make/run.py

```diff
@@ -15,38 +18,58 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("--port", default="/dev/arduino")
     parser.add_argument("--baudrate", type=int, default=38400)
+    parser.add_argument("--reset-mode", type=str, choices=["rts", "dtr", "none"], default="none")
```

Comment:
This line is more than ~80 characters. Change to one argument per line.

Reply:
Ok, I will split it in multiple lines.